pkg/kapp/logs/pod_log.go

```diff
@@ -76,3 +89,12 @@ func (l PodLog) isWaitingContainer(cont corev1.Container, statuses []corev1.Cont
 	}
 	return false
 }
+
+func (l PodLog) isContainerInSlice(cont corev1.Container, containers []string) bool {
```

Comment:
let's change this to return true if there are 0 containers. otherwise check existence in slice.

Reply:
may be also rename isContainerInSlice to isWatchingContainer